Migrating to to reactive rest client has broken horreum-client

Horreum client tests are now failing with 

```
[ERROR] io.hyperfoil.tools.HorreumClientTest.ConfigQuickstartTest  Time elapsed: 1.414 s  <<< ERROR!
java.lang.NoClassDefFoundError: javax/servlet/ServletContext
    at io.hyperfoil.tools.HorreumClientTest.ConfigQuickstartTest(HorreumClientTest.java:22)
Caused by: java.lang.ClassNotFoundException: javax.servlet.ServletContext
    at io.hyperfoil.tools.HorreumClientTest.ConfigQuickstartTest(HorreumClientTest.java:22)
```

The horreum client can not be easily updated to use the reactive-rest client as `horreum-client` depends on ;

```
        <dependency>
            <groupId>org.keycloak</groupId>
            <artifactId>keycloak-admin-client</artifactId>
        </dependency>
```

which in-turn depends on;

```
        <dependency>
            <groupId>org.jboss.resteasy</groupId>
            <artifactId>resteasy-client</artifactId>
        </dependency>
```

--- a/horreum-backend/pom.xml
+++ b/horreum-backend/pom.xml
@@ -105,11 +105,11 @@
         </dependency>
         <dependency>
             <groupId>io.quarkus</groupId>
-            <artifactId>quarkus-rest-client-reactive</artifactId>
+            <artifactId>quarkus-rest-client</artifactId>
         </dependency>
         <dependency>
             <groupId>io.quarkus</groupId>
-            <artifactId>quarkus-rest-client-reactive-jackson</artifactId>
+            <artifactId>quarkus-rest-client-jackson</artifactId>
         </dependency>
         <dependency>
             <groupId>org.apache.commons</groupId>

